.github/workflows/go-test-config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip32bit": true
```

Comment:
why 32-bit tests are skipped?

Reply:
It seems to be set like that across all out repos

Reply:
I.e. https://github.com/ipni/storetheindex/blob/main/.github/workflows/go-test-config.json#L2

Reply:
The reason it is set in those repose is because they depend on Pebble. 

Does this repo _need_ to skip 32 bit tests?

Reply:
No, I'll remove